client/app/components/group/_pickupList/pickupList.controller.js

```diff
@@ -191,6 +191,23 @@ class PickupListController {
       }
     });
   }
+
+  joinPickup(pickupId) {
```

Comment:
I came to the conclusion that it's good to have this here (and not move it to pickupListItem). So we can leave it.